dynamiqs/utils/operators.py

```diff
@@ -24,44 +24,93 @@
 
 @complex_tensor
 def sigmax(*, dtype=None, device=None) -> Tensor:
-    """Pauli $X$ operator."""
+    """Returns the Pauli $X$ operator.
+
+    Args:
+        dtype (torch.dtype): Data type of the returned tensor. Defaults to the complex
+            data type specified by `torch.get_default_dtype()`.
+        device (torch.device): Device of the returned tensor. Defaults to the globally
+            defined default device.
```

Comment:
```suggestion
        dtype: Data type of the returned tensor.
        device: Device of the returned tensor.
```
I'd suggest shortening the doc to this in order not to overload it, we can write smwh clearly in a "must read" section what all dtype and device mean and how they default, instead of repeating these two lines everywhere.